docs/bundles/plain.md

```diff
@@ -29,16 +60,125 @@ manifests
 └── deployment.yaml
 ```
 
+For a bundle git repo, any directory that contains only static Kubernetes manifests checked into a git repository
+accessible via a remote URL can be considered a plain bundle and sourced by the plain provisioner. For more information
+on bundles backed by git repositories, see the [git based bundles doc](git-bundles.md).
+
 > Note: there must be at least one resource in the manifests directory in order for the bundle to be a valid
 > plain+v0 bundle.
```

Comment:
```suggestion
> `plain+v0` bundle.
```

Reply:
addressed in https://github.com/operator-framework/rukpak/pull/642/commits/486faf8915e2f3f891ee769b99651e3039d052b1